app/models/validator_block_history.rb

```diff
@@ -45,4 +51,16 @@ def self.median_skipped_slot_percent_for(network, batch_uuid)
       batch_uuid: batch_uuid
     ).median(:skipped_slot_percent)
```

Comment:
Let's calculate the median moving average instead of :skipped_slot_percent. Same for self.average_skipped_slot_percent_for. We will calculate the scores from the MA, not the instant values. You'll need to chase this through `app/logic/validator_score_v1_logic.rb` too.